crates/store/src/store.rs

```diff
@@ -146,17 +101,17 @@ impl StoreHandle {
     ///
     /// # Errors
     ///
-    /// Returns [`StoreError::InvalidArgument`] if the entry has been deleted or compacted.
-    pub fn get_length<S: Storage>(&self, store: &Store<S>) -> StoreResult<usize> {
+    /// Returns an error if the entry has been deleted or compacted.
```

Comment:
Let's keep the constant name here since it will be exposed. Same throughout the file.